### DISCUSSION TITLE
How to ssh or jump directly inside the sandbox without --join?

### DISCUSSION THREAD

**Discussion by osevan** [netblue30/firejail#6132] [Q&A] [answered]
Hi community, 

I want give someone ssh access to my computer,  but inside Firejailed Sandbox with --private mode on, and Overlayfs

How i can  let someone connect inside Firejailed environment? 

Im open for socat sockets piping  solution from outside of firejail .

So every help is useful

Thanks and

Best regards

**Comment by rusty-snake** [comment, marked as answer]
https://zyoung.dev/project/ssh-chroot-using-firejail-as-login-shell/

**Reply by osevan** [reply]
Big thx answered everything.

**Comment by osevan** [comment]
I have a solution for other guys too:

Starting firejail --private and starting inside tmate -F

This will grant access to inside shell over tmate server and you can share session with your friends co coworker.
